worker/src/RTC/RtpDictionaries/RtpCodecParameters.cpp

```diff
@@ -24,9 +24,10 @@ namespace RTC
 		this->clockRate = data->clockRate();
 
 		// channels is optional.
-		if (data->channels().has_value())
+		auto channels = data->channels();
+		if (channels.has_value())
```

Comment:
What is going on in this PR? We always add blank lines after variable declaration and before `if` conditions and before `return`.


```suggestion
		auto channels = data->channels();

		if (channels.has_value())
```

Reply:
Fine